chart/kubeapps/README.md

```diff
@@ -181,232 +181,232 @@ Once you have installed Kubeapps follow the [Getting Started Guide](https://gith
-| `dashboard.nodeAffinityPreset.key`                | Node label key to match. Ignored if `affinity` is set                                        | `""`                 |
-| `dashboard.nodeAffinityPreset.values`             | Node label values to match. Ignored if `affinity` is set                                     | `[]`                 |
-| `dashboard.affinity`                              | Affinity for pod assignment                                                                  | `{}`                 |
-| `dashboard.nodeSelector`                          | Node labels for pod assignment                                                               | `{}`                 |
-| `dashboard.tolerations`                           | Tolerations for pod assignment                                                               | `[]`                 |
-| `dashboard.priorityClassName`                     | Priority class name for Dashboard pods                                                       | `""`                 |
-| `dashboard.hostAliases`                           | Custom host aliases for Dashboard pods                                                       | `[]`                 |
-| `dashboard.extraVolumes`                          | Optionally specify extra list of additional volumes for Dasbhoard pods                       | `[]`                 |
-| `dashboard.extraVolumeMounts`                     | Optionally specify extra list of additional volumeMounts for Dasbhoard container(s)          | `[]`                 |
-| `dashboard.sidecars`                              | Add additional sidecar containers to the Dasbhoard pod                                       | `[]`                 |
-| `dashboard.initContainers`                        | Add additional init containers to the Dasbhoard pods                                         | `[]`                 |
-| `dashboard.service.port`                          | Dasbhoard service HTTP port                                                                  | `8080`               |
-| `dashboard.service.annotations`                   | Additional custom annotations for Dasbhoard service                                          | `{}`                 |
+| Name                                              | Description                                                                                  | Value                        |
+| ------------------------------------------------- | -------------------------------------------------------------------------------------------- | ---------------------------- |
+| `dashboard.image.registry`                        | Dashboard image registry                                                                     | `docker.io`                  |
+| `dashboard.image.repository`                      | Dashboard image repository                                                                   | `bitnami/kubeapps-dashboard` |
```

Comment:
We don't normally manually update all the bitnami chart images to kubeapps ones in the README do we? I assume this is just because we'd run the readme generator on our main branch (if so, I assume we could run it as part of this PR generation, given that the values.yaml has the correct tags etc.?)

Reply:
This PR is going to be stale before the next upstream change happens :P Any reason not to land it? Thanks for creating the issue Antonio.

Reply:
Ah - or I can manually run the generator too - let me do that.

Reply:
Ooops, I just forgot to add the +1 in GitHub; sorry!
However, let me run the readmenator to avoid having `bitnami/xxx` images in our dev chart docs and I'll merge it straight away